lib/shared/types.js

```diff
@@ -25,6 +25,14 @@ module.exports = {};
  * @property {"script"|"module"} [sourceType] The source code type.
  */
 
+/**
+ * @typedef {Object} LanguageOptions
+ * @property {number|"latest"} [ecmaVersion] The ECMAScript version (or revision number).
+ * @property {"script"|"module"} [sourceType] The source code type.
+ * @property {string|Object} [parser] The parser to use.
+ * @property {Object} [parserOptions] The parser options to use.
```

Comment:
This typedef is missing `LanguageOptions.globals` property.

Reply:
Nice catch. Added.